docs/guide/notable-features.md

```diff
@@ -22,15 +22,15 @@ Rolldown does not polyfill Node built-ins when targeting the browser. You can op
 ## Built-in transforms
 
 Rolldown supports the following transforms out of the box, powered by [Oxc](https://oxc.rs/docs/guide/usage/transformer).
-The transform is configurable via the [`transform`](/apis/config-options#transform) option.
+The transform is configurable via the `transform` option.
```

Comment:
Is there no new link here?

Reply:
No. `transform` page is missing currently.